src/main/java/ee/buerokratt/ruuter/helper/ConfigurationMappingHelper.java

```diff
@@ -57,6 +59,9 @@ private Map<String, ConfigurationStep> convertNodeMapToStepMap(Map<String, JsonN
 
     private ConfigurationStep convertJsonNodeToConfigurationStep(JsonNode jsonNode) throws JsonProcessingException {
         if (jsonNode.get("call") != null) {
+            if (Objects.equals(mapper.convertValue(jsonNode.get("call"), String.class), "reflect.mock")) {
```

Comment:
A better solution would be: `jsonNode.get("call").asText().equals("reflect.mock")`